cognite/transformations_cli/commands/deploy/transformation_types_legacy.py

```diff
@@ -85,7 +85,7 @@ class TransformationConfigLegacy:
     authentication: Union[AuthConfigLegacy, ReadWriteAuthConfigLegacy, None]
     api_key: Union[str, ReadWriteApiKeyLegacy, None]
     schedule: Optional[str]
-    destination: DestinationLegacy
+    destination: Union[str, DestinationLegacy]
```

Comment:
This was wrong, we accepted string for flat types such as assets, timeseries etc.